src/djcytoscape/forms.py

```diff
@@ -1,28 +1,21 @@
 from django import forms
-
-from queryset_sequence import QuerySetSequence
+from django.contrib.contenttypes.models import ContentType
 
 from utilities.forms import FutureModelForm
-from utilities.fields import ContentObjectChoiceField
-from utilities.widgets import ContentObjectSelect2Widget
+from utilities.fields import AllowedContentObjectChoiceField as ContentObjectChoiceField
```

Comment:
Please don't `import as` unless it's a standard convenion, it makes the code hard to maintain and read.  Explicit is better than Implicit!  Now there are two AllowedContentObjectChoice field classes.  Give the class below a new name that fits why you are extending it.